data/pubspec.yaml

```diff
@@ -33,6 +33,11 @@ dependencies:
   # flutter_uploader
   flutter_uploader: ^1.2.0
 
+  # flutter_downloader
+  flutter_downloader: ^1.5.2
+
+  path_provider: ^1.6.24
```

Comment:
we already have this library inside FlutterSDK. So I dont think you need to declare it here

Reply:
I don't understand, can you give me more details please ?

Reply:
Oh sorry mb, just keep it here. We still need to declare this library in pubspec